src/pages/products/sgid/transportation/uta-routes.astro

```diff
@@ -27,7 +27,7 @@ export const metadata: IMetadata = {
 
 const page: IPageMetadata = {
   ...metadata,
-  application: 'https://rideuta.maps.arcgis.com/apps/webappviewer/index.html?id=448c3cfe44264f1e8931073dfb6e5417',
+  application: 'https://maps.rideuta.com/portal/apps/instant/sidebar/index.html?appid=b5db22dc09a24203886607cf3e2abb30',
```

Comment:
**question**: did you also update this in the sgid index?

Reply:
i did now. ha. thanks for the reminder.

Reply:
i could use some help keeping the sgid index current and useful.